test/test.mk

```diff
@@ -24,9 +24,17 @@ test/automated/packaging:
 ifndef NR_LICENSE_KEY
 	@echo "NR_LICENSE_KEY variable must be provided for test/automated/packaging"
 	exit 1
-else
-	@ANSIBLE_DISPLAY_SKIPPED_HOSTS=NO ANSIBLE_DISPLAY_OK_HOSTS=NO ansible-playbook -i $(CURDIR)/test/automated/ansible/inventory.ec2 -e nr_license_key=$(NR_LICENSE_KEY) $(CURDIR)/test/packaging/ansible/test.yml
 endif
+ifndef NEW_RELIC_API_KEY
```

Comment:
where are these env-vars being set?

Reply:
it is in release docs (ohai-docs) This env var should be provided for the test run